src/actions/index.ts

```diff
@@ -59,6 +60,7 @@ class Actions implements ActionRecord {
   replaceWithTarget = new Bring(this.graph);
   reverseTargets = new Reverse(this.graph);
   rewrapWithPairedDelimiter = new Rewrap(this.graph);
+  runCommandOnSelection = new RunCommandOnSelection(this.graph);
```

Comment:
I believe you should add this one to the `Action` type.  Would be nice if there were some way to make it error when you forget to do so 🤔

Also as mentioned in other PR, I think I'd tweak this name